src/App.php

```diff
@@ -234,11 +232,18 @@ public function callExit($from_shutdown = false)
      * @param Throwable $exception
      *
      * @throws \atk4\core\Exception
+     * @throws ExitApplicationException
      *
      * @return bool
      */
     protected function caughtException(Throwable $exception)
     {
+        // if the intent is to exiut from application
+        // catch Exit Exception and return
+        if ($exception instanceof ExitApplicationException) {
+            return true;
+        }
```

Comment:
clever approach to avoid error.

But it's different to what we dicsussed and i think this can cause problem if you run app with "catch_exception = false".

Also i think it will create some insanity if used with logger (which normally log all unhandled exceptions)

Best to implement the property App::call_exit = true , which can be modified to "false" if exception is required.